pages/common/cs-fetch.md

```diff
@@ -0,0 +1,28 @@
+# cs fetch
+
+> Fetch fetches the JARs of one or more dependencies.
+> More information: <https://get-coursier.io/docs/cli-fetch>.
+
+- Fetch a specific version of a jar:
+
+`cs fetch {{groupId}}:{{artifactId}}:{{artifactVersion}}`
+
+- Fetch a package and evaluate the classpath corresponding to the selected package in an env var:
+
+`CP="$(cs fetch --classpath org.scalameta::scalafmt-cli:latest.release)"`
+
+- Fetch the source jar:
+
+`cs fetch --sources {{groupId}}:{{artifactId}}:{{artifactVersion}}`
+
+- Fetch the javadoc jars:
+
+`cs fetch --javadoc {{groupId}}:{{artifactId}}:{{artifactVersion}}`
+
+- Fetch dependency with javadoc jars and source jars:
+
+`cs fetch --default=true --sources --javadoc {{groupId}}:{{artifactId}}:{{artifactVersion}}`
```

Comment:
```suggestion
- Fetch dependency with javadoc jars and source jars:

`cs fetch --default={{true}} --sources --javadoc {{group_id}}:{{artifact_id}}:{{artifact_version}}`
```

Almost the same issue as above.
Mandatory fix: unexpected `true` boolean literal, placeholder expected.

Reply:
without the `true` the behaviour is exactly the previous one, so it is the only meaningful value to insert. If you don't do that it would not make any sense to specify the ``--default`` flat